.claude/skills/commit.md

```
@@ -0,0 +1,116 @@
+---
+
+## Phase 1 — 분석 및 계획 수립
+
+1. `git status`와 `git diff --cached`로 스테이징된 변경 확인
+2. **secret guard**: `.env`, `password`, `api_key`, `secret`, `token` 패턴 검색 — 발견 시 즉시 중단
+3. 변경된 파일들을 도메인/기능/계층 단위로 커밋 그룹 분류
+4. 아래 형식의 계획표를 출력하고 **사용자 승인 대기**
+
+### 계획표 출력 형식
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)